control-main.go

```diff
@@ -0,0 +1,132 @@
+/*
+ * Minio Cloud Storage, (C) 2016 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"net/rpc"
+	"net/url"
+
+	"github.com/minio/cli"
+)
+
+// "minio control" command.
+var controlCmd = cli.Command{
+	Name:   "control",
+	Usage:  "minio control commands.",
+	Action: mainControl,
+	Subcommands: []cli.Command{
+		healCmd,
+	},
+}
+
+func mainControl(c *cli.Context) {
+	cli.ShowCommandHelp(c, "")
+}
+
+var healCmd = cli.Command{
+	Name:   "heal",
+	Usage:  "To heal objects.",
+	Action: healControl,
+	CustomHelpTemplate: `NAME:
+  minio {{.Name}} - {{.Usage}}
+
+USAGE:
+  minio {{.Name}} [OPTIONS] URL
+
+EAMPLES:
+  1. Heal an object.
+     $ minio control heal http://localhost:9000/songs/classical/western/piano.mp3
+
+  2. Heal all objects in a bucket recursively.
+     $ minio control heal http://localhost:9000/songs
+
+  3. Heall all objects with a given prefix recursively.
+     $ minio control heal http://localhost:9000/songs/classical/
+`,
+}
+
+// "minio control heal" entry point.
+func healControl(c *cli.Context) {
+	// Parse bucket and object from url.URL.Path
+	parseBucketObject := func(path string) (bucket string, object string) {
+		path = path[1:]
```

Comment:
I do a check:
```
89         if path == "" || path == slashSeparator {                                                                                                                            
90                 cli.ShowCommandHelpAndExit(c, "heal", 1)                                                                                                                     
91         }     
```
but it's cleaner and selfcontained i.e doesn't depend on this check.